wasm.md

```diff
@@ -197,37 +197,53 @@ A `UnOp` operator always produces a result of the same type as its operand.
 
 ```k
     syntax UnOp ::= IUnOp
- //               | FUnOp
  // ---------------------
 
     syntax Instr ::= "(" IValType "." IUnOp ")" | "(" IValType "." IUnOp Instr ")" | IValType "." IUnOp Int
- //                | "(" FValType "." FUnOp ")" | "(" FValType "." FUnOp Instr ")" | FValType "." FUnOp Float
  // ---------------------------------------------------------------------------------------------------------
     rule <k> ( ITYPE . UOP:IUnOp I:Instr ) => I ~> ( ITYPE . UOP ) ... </k>
 
     rule <k> ( ITYPE . UOP:IUnOp ) => ITYPE . UOP C1 ... </k>
          <valstack> < ITYPE > C1 : VALSTACK => VALSTACK </valstack>
 ```
 
+**TODO**: Unimplemented.
```

Comment:
Perhaps add line:
"Floating point operations" or similar. May not be immediately obvious to a reader what the "F" stands for.